psi/build.gradle.kts

```diff
@@ -25,13 +25,21 @@
  */
 
 import io.spine.internal.dependency.IntelliJ
+import io.spine.internal.dependency.Spine
 
 repositories {
     intellijReleases
     jetBrainsCacheRedirector
 }
 
 dependencies {
-    api(IntelliJ.Platform.core)
-    api(IntelliJ.Platform.util)
+    api(Spine.base)
+    IntelliJ.Platform.run {
+        arrayOf(
+            core,
+            util,
+            coreImpl,
+            codeStyle
+        ).forEach { api(it) }
```

Comment:
Some of them are probably `impl`. But let's leave it `api` for now, until we start using our `psi` API with ProtoData or other libraries.